x-pack/plugins/triggers_actions_ui/public/plugin.ts

```diff
@@ -99,27 +110,19 @@ export class Plugin
           unknown
         ];
 
-        const { boot } = await import('./application/boot');
+        const { renderApp } = await import('./application/app');
         const kibanaFeatures = await pluginsStart.features.getFeatures();
-        return boot({
+        return renderApp({
           dataPlugin: pluginsStart.data,
           charts: pluginsStart.charts,
           alerts: pluginsStart.alerts,
           element: params.element,
-          toastNotifications: coreStart.notifications.toasts,
-          http: coreStart.http,
-          uiSettings: coreStart.uiSettings,
-          docLinks: coreStart.docLinks,
-          chrome: coreStart.chrome,
-          savedObjects: coreStart.savedObjects.client,
-          I18nContext: coreStart.i18n.Context,
-          capabilities: coreStart.application.capabilities,
-          navigateToApp: coreStart.application.navigateToApp,
           setBreadcrumbs: params.setBreadcrumbs,
           history: params.history,
           actionTypeRegistry,
           alertTypeRegistry,
           kibanaFeatures,
+          ...coreStart,
```

Comment:
**optional nit:** I would move this to the top. It would follow the type inheritance if ever something here overwrites a `coreStart` property.